src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -197,8 +199,7 @@ public static class HttpMessageHandlerIntegration
             var headers = request.GetProperty<object>("Headers").GetValueOrDefault();
             var reportedType = callOpCode == OpCodeValue.Call ? httpClientHandler : handler.GetType();
 
-            if (!(reportedType.FullName.Equals(HttpClientHandler, StringComparison.OrdinalIgnoreCase) || IsSocketsHttpHandlerEnabled(reportedType)) ||
-                !IsTracingEnabled(headers))
+            if (!IsTracingEnabled(headers))
```

Comment:
We don't need to type checks here?

Reply:
I don't think so. We're in HttpClientHandler_SendAsync so we know for sure this is a HttpClientHandler

Reply:
(and SocketsHttpHandler doesn't inherit from HttpClientHandler so that one is out as well)

Reply:
Makes sense, thanks.